.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]<Enter comprehensive title>"
+labels: bug
+assignees: ''
+
```

Comment:
printer type: {MK3S, MK3, MK2.5S, MK2.5, MK2S, MK2}
upgrade: {MMU2S, MMU2, MMU1}
firmware version: {3.8.1, 3.8.1 RC1, ...}

---
As a summary, please make sure you have ticked all points on this
checklist:
-->
* [ ] I searched in open and closed issues and couldn't find anything that matches.
* [ ] Printer type (MK3S, MK3, MK2.5S, MK2.5, MK2S, MK2)
* [ ] Printer firmware version (3.8.1, 3.8.1-RC1, ...)
* [ ] Upgrade (MMU2S, MMU2, MMU1)
* [ ] Upgrade firmware version (1.0.6, 1.0.6-RC2, ...)